casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -293,19 +291,27 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
       deploy: DeployData,
       reducer: ChargingReducer[F],
       errorLog: ErrorLog[F]
-  )(implicit C: _cost[F]) = {
-    import coop.rchain.catscontrib.mtl.implicits._
+  )(implicit C: _cost[F]): F[EvaluateResult] = {
     implicit val rand: Blake2b512Random = Blake2b512Random(
       DeployData.toByteArray(ProtoUtil.stripDeployData(deploy))
     )
-    Interpreter[F].injAttempt(
-      reducer,
-      errorLog,
-      deploy.term,
-      Cost(deploy.phloLimit)
-    )
+    val intrinsicGas: Long = parsingCost(deploy.term).value
```

Comment:
Why not just `parsingCost` or `parsingGas`?

Reply:
In the future, it's possible that it includes costs besides parsing. For example, while we don't have charging in the normalizer, we might include a cost for normalization.

Reply:
I see; I say we rename it then

Reply:
Rename it to what? The terminology is taken from Ethereum, which has a somewhat analogous minimum.

Reply:
To intrinsic gas